test/powershell/Host/Startup.Tests.ps1

```diff
@@ -103,7 +103,7 @@ Describe "Validate start of console host" -Tag CI {
     }
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Test PSDesiredStateConfiguration.Error occurred in Describe block
```powershell
The term 'Get-Platform' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1:31 char:21
```